src/constants.ts

```diff
@@ -4,5 +4,5 @@
  */
 export const DEFAULT_BLOCKCHAIN_IMAGE = 'fairdatasociety/fdp-contracts-blockchain'
 export const DEFAULT_BLOCKCHAIN_CONTAINER = 'fdp-play-blockchain'
-export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs:v0.7.3'
+export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs:latest'
```

Comment:
latest is not necessary and it only makes the image name long
```suggestion
export const DEFAULT_FAIROS_IMAGE = 'fairdatasociety/fairos-dfs'
```